libtransmission/version.h.in

```diff
@@ -7,7 +7,7 @@
 #define SHORT_VERSION_STRING      "${TR_USER_AGENT_PREFIX}"
 #define LONG_VERSION_STRING       "${TR_USER_AGENT_PREFIX} (${TR_VCS_REVISION})"
 #define VERSION_STRING_INFOPLIST  ${TR_USER_AGENT_PREFIX}
-#define BUILD_STRING_INFOPLIST    14714.${TR_VERSION_MAJOR}.${TR_VERSION_MINOR}.${TR_VERSION_PATCH}
+#define BUILD_STRING_INFOPLIST    ${TR_LIB_VERSION_MAJOR}.${TR_LIB_VERSION_MINOR}.${TR_LIB_VERSION_PATCH}
```

Comment:
This line doesn't seem to be used anymore -- can it be removed?

Reply:
This line is the critical one when building with cmake: it defines BUILD_STRING_INFOPLIST, which is used for CFBundleVersion in `macosx/info.plist`